src/Interfaces.jl

```diff
@@ -1777,6 +1777,14 @@ function Base.:*(b::PVector,a::Number)
   a*b
 end
 
+function Base.:/(a::Number,b::PVector)
```

Comment:
I would say that this is not correct.

Reply:
Removed this operation in 4036cb268ab832c63b9c41ebb4dcb93690e423c3